evap/evaluation/templates/log/changed_fields_entry.html

```diff
@@ -0,0 +1,35 @@
+{% load static %}
+{% load i18n %}
+
+{{ message }}
+
+<ul>
+    {% for field_name, actions in fields.items %}
+        {% for action in actions %}
+            <li>
+                {% if action.type == "add" %}
+                    {{ action.label}} {% trans "added" %}: {{ action.items|join:", " }}
+                {% elif action.type == "remove" %}
+                    {{ action.label}} {% trans "removed" %}: {{ action.items|join:", " }}
+                {% elif action.type == "clear" %}
```

Comment:
This mode is not in evaluation/models.py:204 and I didn't read anything about a "clear" action anywhere so far... Did I miss something?

Reply:
'clear' is used when m2m sets are cleared. A m2m signal exists, but I have not seen it happen in evap yet.